HeartbeatLoggingTestUtils/Sources/HeartbeatLoggingTestUtils.swift

```diff
@@ -13,10 +13,12 @@
 // limitations under the License.
 
 import Foundation
-import XCTest
+import XCTest // COMPILE ERROR: No such module 'XCTest'
```

Comment:
I'm not sure what why this is happening. I looked at how popular swift testing cocoapods do it and they specify `XCTest` as a framework.

https://github.com/firebase/firebase-ios-sdk/blob/4e9c9a907ddaf41d3590041b5d2c6b688389de04/HeartbeatLoggingTestingUtils.podspec#L37

https://github.com/firebase/firebase-ios-sdk/blob/4e9c9a907ddaf41d3590041b5d2c6b688389de04/HeartbeatLoggingTestingUtils.podspec#L48-L51

Reply:
And https://github.com/erikdoe/ocmock/blob/master/OCMock.podspec.